var/spack/repos/builtin/packages/gmsh/gmsh-4.7.1-cgns-type.patch

```diff
@@ -0,0 +1,1006 @@
+From edf973c92d574ac91cb31e6a933be38113b8f78d Mon Sep 17 00:00:00 2001
```

Comment:
You can delete this file now.

Reply:
Thanks for your reminder. The file has been deleted.